### ISSUE TITLE
Unexpected rewrite set operation type

### ISSUE THREAD

**Issue by TheYkk** [authorizer-tech/access-controller#21]
```sh
INFO[0000] Starting access-controller                   
INFO[0000]   Version: dev                               
INFO[0000]   Date: unknown                              
INFO[0000]   Commit: none                               
INFO[0000]   Go version: go1.16.5                       
INFO[0000] Cluster member with id '8804a89f-5ef4-4a2d-b4dd-e97f6d7c527e' joined the cluster at address '192.168.1.33:7946' 
INFO[0000] Starting grpc server at ':50052'..           
INFO[0000] Starting grpc-gateway server at ':8082'..    
panic: Unexpected rewrite set operation type

goroutine 271 [running]:
github.com/authorizer-tech/access-controller/internal.validateRewriteRelations(0xc000475ad0, 0xc0004e31c0, 0xc00001f5e0, 0xc0004e31c0)
        /home/kaan/Work/projects/ope-source/access-controller/internal/access-controller.go:1263 +0x425
github.com/authorizer-tech/access-controller/internal.(*AccessController).WriteConfig(0xc0001ed860, 0xe35520, 0xc000475a70, 0xc000475aa0, 0xc0001ed860, 0xc000475a70, 0xc000087ba0)
        /home/kaan/Work/projects/ope-source/access-controller/internal/access-controller.go:933 +0x16d
github.com/authorizer-tech/access-controller/genprotos/authorizer/accesscontroller/v1alpha1._NamespaceConfigService_WriteConfig_Handler(0xd1cfa0, 0xc0001ed860, 0xe35520, 0xc000475a70, 0xc00010eba0, 0x0, 0xe35520, 0xc000475a70, 0xc00012e900, 0x1a)
        /home/kaan/Work/projects/ope-source/access-controller/genprotos/authorizer/accesscontroller/v1alpha1/namespace_service_grpc.pb.go:106 +0x214
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0002b8380, 0xe3ddd8, 0xc000682900, 0xc0000d0c60, 0xc000539e60, 0x12c08a0, 0x0, 0x0, 0x0)
        /home/kaan/go/pkg/mod/google.golang.org/grpc@v1.37.0-dev.0.20210309003715-fce74a94bdff/server.go:1217 +0x52b
google.golang.org/grpc.(*Server).handleStream(0xc0002b8380, 0xe3ddd8, 0xc000682900, 0xc0000d0c60, 0x0)
        /home/kaan/go/pkg/mod/google.golang.org/grpc@v1.37.0-dev.0.20210309003715-fce74a94bdff/server.go:1540 +0xd0c
google.golang.org/grpc.(*Server).serveStreams.func1.2(0xc000116060, 0xc0002b8380, 0xe3ddd8, 0xc000682900, 0xc0000d0c60)
        /home/kaan/go/pkg/mod/google.golang.org/grpc@v1.37.0-dev.0.20210309003715-fce74a94bdff/server.go:878 +0xab
created by google.golang.org/grpc.(*Server).serveStreams.func1
        /home/kaan/go/pkg/mod/google.golang.org/grpc@v1.37.0-dev.0.20210309003715-fce74a94bdff/server.go:876 +0x1fd
exit status 2

```
It shouldn't panic 

![image](https://user-images.githubusercontent.com/21688838/122044011-a9cdb380-cde4-11eb-8c33-1631f1a33371.png)

**Comment by jon-whit** [comment]
I've got a big release coming up that fixes this. Hang tight, it's coming tonight 😉

Thanks for taking a look and reporting the issue 👍🏻

**Comment by jon-whit** [comment]
👀 taking a look. Also, I'm going to be releasing an initial v0.1.0 release this evening that includes some bug fixes along with a handful of other great things.

Keep your eyes open for the release 😉!

**Comment by jon-whit** [comment]
@TheYkk it looks like you have a slight typo in the configuration. If you use `this: {}` instead of `_this: {}` it works great.

But you bring up a good point. This should fail input validation because of the issue. It shouldn't panic. Working on a fix as we speak 👍

**Comment by TheYkk** [comment]
I test with your example json files

**Comment by jon-whit** [comment]
> I test with your example json files

Ah, I see. Those are a tad out of date I think. At one point I was doing namespace configurations via files, but now it's via API. It think they became out of date in that transition. Let me update them so that people can still use them as a starting point for HTTP calls like you're doing. Thanks for the feedback 👍

**Comment by jon-whit** [comment]
#23 will address the input validation issue. I'll tackle that in the next day or so.

Thanks for the heads up!

**Comment by jon-whit** [comment]
@TheYkk this should now be fixed up in `v0.1.1` release. I added input request validation in this release, and so now you should get an appropriate error message if you encounter this error again.
